posthog/api/decide.py

```diff
@@ -12,21 +14,73 @@
 from posthog.utils import cors_response
 
 
-def _load_data(data: str) -> Dict[str, Any]:
+def _load_data(request) -> Optional[Dict[str, Any]]:
```

Comment:
This function is the exact same as in capture.py. Could be worth separating it out into utils.py

Reply:
Done!